packages/runtime/lib/runtime.js

```diff
@@ -1150,7 +1158,8 @@ class Runtime extends EventEmitter {
 
     const execArgv = []
 
-    if (!serviceConfig.isPLTService && config.telemetry && config.telemetry.enabled !== false) {
+    // TODO@PI: Replace me with the option for 22
+    if (!serviceConfig.skipTelemetryHooks && config.telemetry && config.telemetry.enabled !== false) {
```

Comment:
Is this set correctly for composer and db too?

Reply:
Yup, it is!